text/dotnet-core/0007-dependencies.md

```diff
@@ -0,0 +1,70 @@
+# Decide Which .NET Dependencies Will Be Paketo-hosted
+
+## Proposal
+
+The following dependencies should be kept as Paketo-hosted dependencies:
+* [ICU](https://github.com/paketo-buildpacks/icu/blob/main/buildpack.toml)
+
+
+The following dependencies should be removed as Paketo-hosted dependencies:
+* [.NET Core SDK](https://github.com/paketo-buildpacks/dotnet-core-sdk/blob/main/buildpack.toml)
+
+The following will be a new dependencies that are not currently Paketo-hosted
+and should not be added:
+* .NET Core ASP.NET Core Runtime
+* [vsdbg](https://github.com/paketo-buildpacks/vsdbg/blob/main/buildpack.toml)
+
+## Rationale
+
+### ICU
+
+Keep this as a Paketo-hosted dependency.
+
+ICU currently [only builds for one version of
+Ubuntu](https://github.com/unicode-org/icu/releases/tag/release-71-1) therefore
+in order to support a wide range of stacks we should continue to compile and
+host this independent of upstream. The current build process can be seen
+[here](https://github.com/cloudfoundry/buildpacks-ci/blob/7c76257aa285bf148767a14dd62953b1f0b163b0/tasks/build-binary-new/builder.rb#L462).
```

Comment:
I think you're probably right about continuing to compile it. 

I am wondering though if we are doing something strange by compiling all new versions of ICU for different stacks. Would a user of ICU in the wild even want/be able to use a new version of ICU on an old OS? 

Also fwiw, it does look like ICU dropped a Jammy Jellyfish-compatible version in June 2022: https://launchpad.net/ubuntu/+source/icu (70.1 version line)

Reply:
Since we currently make new versions of ICU available on Bionic, I think we should continue doing that until Bionic goes out of support. (That way, we don't have to downgrade the available versions of ICU.) It's worth investigating whether users will still expect the latest ICU versions on Bionic when there's another LTS version of Ubuntu available.